css/main.css

```diff
@@ -32,6 +32,19 @@ nav .btn-login {
   float: right;
 }
 
+nav .button{
+  font-size: 20px;
+  border-radius: 06px;
```

Comment:
`6px` would be more readable than `06px`

Reply:
ok